telepresence/outbound/cidr.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2018 Datawire. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ipaddress
+import json
+from itertools import chain
+from subprocess import CalledProcessError
+from typing import List
+
+from telepresence.proxy import RemoteInfo
+from telepresence.runner import Runner
+from telepresence.utilities import random_name
+
+
+def covering_cidr(ips: List[str]) -> List[str]:
+    """
+    Given list of IPs, return CIDR that covers them all.
+
+    Presumes it's at least a /24.
+    """
```

Comment:
Please update the function name and doc string.

Reply:
Done